.github/workflows/pythonrelease.yml

```diff
@@ -16,11 +16,12 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install --upgrade --force-reinstall twine pep517
+        python -m pip install --upgrade --force-reinstall pep517
```

Comment:
You want to install twine. And use the build project instead of pep517 - https://github.com/tox-dev/tox/blob/rewrite/.github/workflows/check.yml#L108-L129

https://github.com/pypa/build

Reply:
Okay, I'll switch to `build`. I was gonna not bother since `main` has that change already has this change from you.
I just wanted something that would work while we finish the v4 rewrite in case we needed to release again